Makefile

```diff
@@ -26,10 +26,10 @@ docker-compose-build:
 
 .PHONY: review
 review: test-cluster ## Specify review AKS environment
-	# PULL_REQUEST_NUMBER is set by the GitHub action
-	$(if $(PULL_REQUEST_NUMBER), , $(error Missing environment variable "PULL_REQUEST_NUMBER"))
+	# PR_NUMBER is set by the GitHub action
```

Comment:
https://github.com/DFE-Digital/npq-registration/blob/main/.github/actions/deploy-environment-to-aks/action.yml#L59 is passing PULL_REQUEST_NUMBER here and L74